From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: Add upstream patch

---------8<----------
From: 'Hartmut&nbsp;Holzgraefe'Date: December 14 2006 6:07pm
Subject: bk commit into 5.1 tree (hartmut:1.2359) BUG#25075

Below is the list of changes that have just been committed into a local
5.1 repository of hartmut. When hartmut does a push these changes will
be propagated to the main repository and, within 24 hours after the
push, to the public repository.
For information on how to access the public repository
see http://dev.mysql.com/doc/mysql/en/installing-source-tree.html

ChangeSet@stripped, 2006-12-14 18:06:48+01:00, hartmut@stripped +1 -0
  Don't init query cache if its not compiled in (Bug #25075)

  sql/sql_class.cc@stripped, 2006-12-14 18:06:43+01:00, hartmut@stripped +2 -0
    Don't init query cache if its not compiled in (Bug #25075)

# This is a BitKeeper patch.  What follows are the unified diffs for the
# set of deltas contained in the patch.  The rest of the patch, the part
# that BitKeeper cares about, is below these diffs.
# User:	hartmut
# Host:	linux.site
# Root:	/home/hartmut/projects/mysql/dev/5.1
---------8<----------

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---

--- 1.304/sql/sql_class.cc	2006-12-14 18:07:00 +01:00
+++ 1.305/sql/sql_class.cc	2006-12-14 18:07:00 +01:00
@@ -262,7 +262,9 @@
 #endif
   client_capabilities= 0;                       // minimalistic client
   net.last_error[0]=0;                          // If error on boot
+#ifdef HAVE_QUERY_CACHE
   query_cache_init_query(&net);                 // If error on boot
+#endif
   ull=0;
   system_thread= NON_SYSTEM_THREAD;
   cleanup_done= abort_on_warning= no_warnings_for_error= 0;
